d2/src/main/java/com/linkedin/d2/balancer/LoadBalancer.java

```diff
@@ -84,6 +87,12 @@ default void getLoadBalancedServiceProperties(String serviceName, Callback<Servi
     }
   }
 
+  default void getLoadBalancedClusterProperties(String clusterName,
```

Comment:
"getLoadBalancedClusterAndUriProperties" to be more descriptive? same applies to the other load balancer classes in this PR.